docs/syntax-reference.md

```diff
@@ -679,11 +678,22 @@ conventions, which are slightly different to the syntax of the SET statement abo
 $ cat ksql.properties
 auto.offset.reset=earliest
 
-# Start KSQL in standalone mode with the custom properties above
-$ ksql-cli local --properties-file ./ksql.properties
+# Start a KSQL cli session with the custom properties above
+$ ksql http://localhost:8080 --properties-file ./ksql.properties
+```
+
+Finally, you can provide these configurations to the KSQL server in its properties file. When configuring Kafka Streams
+and Kafka via the server properties file, you must prefix the config names with `streams`. The KSQL server will
```

Comment:
This is not correct. You don't have to prefix the properties with `streams` it will work if you just have the standard config names